pages/linux/grub-install.md

```diff
@@ -0,0 +1,12 @@
+# grub-install
+
+> Install GRUB to a device.
+> More information: <https://www.gnu.org/software/grub/manual/grub/html_node/Installing-GRUB-using-grub_002dinstall.html>
+
+- Install GRUB on a BIOS system:
+
+`grub-install --target=i386-pc /dev/sd{{X}}`
```

Comment:
Did you get `X` from PR #5036 :smile:?  I think se should do this:
```suggestion
`grub-install --target={{i386-pc}} {{/dev/sdX}}`
```

Reply:
I was thinking that since the X is the only part that changes, I'll only give that a token status.

Is anything else than `i386-pc` in active use for BIOS systems?

Reply:
I don't know&mdash;we usually do this with user-passed values.

Reply:
Alright. Someone can improve it in the future if they think that those values are practically static.

Reply:
Actually, if we look at /pages/common/dd.md, it sets the precedent of `/dev/{{sdX}}`

From dd.md: `dd if={{file.iso}} of=/dev/{{usb_drive}} status=progress`

Reply:
There's actually ongoing discussion about this over at #5036, so let's just keep what we have here and when #5036 gets resolved we can fix everything that depends on it, including this.

Reply:
I think we should review the token syntax in the `dd` page, because the entery path is the argument option and some other argument options should use `{{}}` too.
Using the `mkfs` page: we can use `{{path/to/partition}}` here, because this could be `/dev/nvme0n1` too.

Reply:
Use `{{path/to/partition}}`.  It's more generic.  Anyway we need to get #5036 figured out soon so we can avoid this trouble in the future.

Reply:
Maybe {{path/to/device}} would be better as it doesn't point to any specific partion?

Reply:
Sure!